cityofaustin/atd-vz-data
# Add alert banner indicating when a crash does not meet the Vision Zero program's reporting requirements

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/17323
Closes https://github.com/cityofaustin/atd-data-tech/issues/18564

## Testing
**URL to test:** 
Local data model

**Steps to test:**
1. Create a temp crash record
2. Go to the details page of that crash, the crash diagram section should now have a message rendering as an alert
3. Update the private drive flag to YES, see the new banner at the top
4. Update it back to NO, watch the banner go away
5. Update the coordinates to be outside of Austin
6. The banner will come back this time with a message about being outside of AFP


---
#### Ship list
